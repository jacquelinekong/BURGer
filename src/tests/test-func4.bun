def string ret1(string s){
	return s;
}

string t;
t = ret1("omg");
println(t);

def int ret4(int i){
	return i;
}
int a;
a = ret4(5);
println(a);
