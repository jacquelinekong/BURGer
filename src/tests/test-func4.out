omg
4.25
